public/docs/ts/latest/guide/testing.jade

```
@@ -1,102 +1,1758 @@
+  But they are not the only tests you should write.  
+
+  ### Isolated unit tests
+
+  You can and should write [isolated unit tests](#testing-without-testbed "Testing without the TestBed") 
+  for components, directives, pipes, and services.
+  Isolated unit tests examine an instance of a class all by itself without 
+  any dependence on Angular or any injected values. 
+  The tester creates a test instance of the class with new, supplying fake constructor parameters as needed, and
+  then probes the test instance API surface.
+
+  Isolated tests don't reveal how the class interacts with Angular. 
+  In particular, they can't reveal how a component class interacts with its own template or with other components.
+
+  Those are Angular integration tests, written with the _TestBed_.
+
+  ### Angular Integration Testing with the _TestBed_
+
+  The Angular _TestBed_ consists of the `TestBed` class and some helper functions from `@angular/core/testing`.
+.alert.is-important
+  :marked
+    The Angular _TestBed_ is officially _experimental_ and thus subject to change.
+    Consult the [API reference](../api/core/testing/index/TestBed-class.html) for the latest status.
+:marked
+  The `TestBed` creates an Angular test module &mdash; an `@NgModule` class &mdash;
+  that you configure to produce the module environment for the class you want to test. 
+  You tell the `TestBed` to create an instance of the test class and probe that instance with tests.
+  That's the `TestBed` in a nutshell.
+
+  In practice, you get a fresh `TestBed` instance before each Jasmine `it`.
+  This `TestBed` instance comes pre-configured with a baseline of default providers and declarables (components, directives, and pipes)
+  that almost everyone needs. This chapter tests a browser application so the default includes the `CommonModule` declarables
+  and providers (some of them mocked) from `@angular/platform-browser`.
```

I searched the TestBed implementation and I can't see that it includes any default module (which I guess it does since we don't add any by hand to get the default directives).

Leaving that aside, isn't it `BrowserModule` instead of `CommonModule` ? And if not, the explanation saying "This chapter tests a **browser** application so the default includes the `CommonModule`" is misleading. Also the import would be `@angular/common` and not `platform-browser`.